src/Microsoft.DotNet.Build.Tasks/GenerateAssemblyList.cs

```diff
@@ -31,12 +30,6 @@ public override bool Execute()
                 while ((line = listReader.ReadLine()) != null)
                 {
                     var fileName = Path.GetFileName(line);
-                    if (!processedFileNames.Add(fileName))
```

Comment:
Just to reiterate this is not going to affect the dependency list that is used to generate the RunTests.cmd which can be found here (https://github.com/dotnet/buildtools/blob/8f2ab39bde3d0f2105123ab0c5bba5b8475ea94a/src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets#L192-L197) This is the old legacy assembly list that was used for execution by Helix.